src/BlockTypes/ProductRating.php

```diff
@@ -150,12 +150,7 @@ protected function render( $attributes, $content, $block ) {
 						esc_html( $reviews_count )
 					);
 
-					if ( $is_descendent_of_single_product_block ) {
-						$customer_reviews_count = '<a href="' . esc_url( $product_permalink ) . '">' . $customer_reviews_count . '</a>';
-					} elseif ( $is_descendent_of_single_product_template ) {
-						$product_permalink      = untrailingslashit( $product_permalink );
-						$customer_reviews_count = '<a href="' . esc_url( $product_permalink ) . '#reviews">' . $customer_reviews_count . '</a>';
-					}
+					$customer_reviews_count = '<a class="woocommerce-review-link" rel="nofollow" href="' . esc_url( $product_permalink ) . '#reviews">' . $customer_reviews_count . '</a>';
```

Comment:
Is there a specific reason for adding the `woocommerce-review-link` class? I inspected it and it doesn't seem to apply any styles. Is it intended for 3rd party developers to override the styles?

Reply:
Yes, the javascript that sets the tab "reviews" focused is loaded by WC Core. The JS uses the `a.woocommerce-review-link` selector to trigger the event ([source code](https://github.com/woocommerce/woocommerce/blob/154dd236499d8a440edf3cde712511b56baa8e45/plugins/woocommerce/client/legacy/js/frontend/single-product.js/#L40-L44)).

Reply:
Understood. Thank you for the clarification. 🙌🏻